src/accounts/dto/create-account.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class CreateAccountDto {
+  readonly name: string;
```

Comment:
missing validations.

use class validator to validate if the type of name is string and it should not be empty.